docker/utils/entrypoint-dev.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+# Install project
+pip install --user .[ldap,social] mysqlclient psycopg2
```

Comment:
https://pypi.org/project/psycopg2-binary/ rather? comes without allot of the overhead

Reply:
per your own link:
_The binary package is a practical choice for development and testing but in production it is advised to use the package built from sources._

Reply:
arf, didn't see it was in entrypoint-**dev** ... i don't think it's worth the trouble tho, no?